packages/hdwallet-tallyho/src/tallyho.ts

```diff
@@ -33,10 +33,14 @@ export class TallyHoHDWalletInfo implements core.HDWalletInfo, core.ETHWalletInf
   }
 
   public hasNativeShapeShift(): boolean {
-    // It doesn't... yet?
+    // It doesn't... yet? - never ever never ever ever
```

Comment:
correct, false, it's metamask-like

Reply:
"NativeShapeShift" is (was) a feature where KK can show the details of an old-school ShapeShift "shift" similar to the way it shows details of an erc20 send for ETH but you just get blind signing for other ETH stuff. There was a whole extra protobuf signed by and served from our backend the KK would verify signature of and show.